core/src/main/java/nl/nn/adapterframework/http/WebServiceListener.java

```diff
@@ -48,14 +53,16 @@
  * @author Jaco de Groot
  * @author Niels Meijer
  */
-public class WebServiceListener extends PushingListenerAdapter implements Serializable, HasPhysicalDestination, HasSpecialDefaultValues {
+public class WebServiceListener extends PushingListenerAdapter implements Serializable, HasPhysicalDestination, HasSpecialDefaultValues, IWithParameters {
```

Comment:
Waarom implementeert deze listener nu ook parameters?

Reply:
Iets teveel gekopieerd van een aanpassing waarvoor nog een PR komt. Ik heb het nu verwijderd.

https://github.com/ibissource/iaf/pull/378/commits/2a364f7eb123db8ad0e55ff001973ee4c1cb850b

Overigens waren in de method close() 2 unregister statements verwijderd, welke ik nu weer toegevoegd heb (en TODO comments verwijderd). Ik heb getest door meerdere reloads te doen en de WebServiceListeners bleven goed werken. Ik denk dat mijn aanpassing in ServiceDispatcher dit 'gefixt' heeft.